sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -104,7 +106,10 @@ protected TimeSeriesInsightsClient()
         /// Gets Time Series model settings asynchronously.
         /// </summary>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>The model settings which includes model display name, Time Series Id properties and default type Id with the http response <see cref="Response{TimeSeriesModelSettings}"/>.</returns>
+        /// <returns>
+        /// The model settings which includes model display name, Time Series Id properties and default type Id with the
```

Comment:
is the TimeSeriesModelSettings properties prone to change, if so, the description of what it includes can get out of date. can we describe the purpose instead of what it includes?

Reply:
Model settings represent 3 things. Display name, Time Series Id keys and the type Id. How about rewording to something like:
A <see cref="TimeSeriesModelSettings"/> object.

This way, when I user clicks on the TimeSeriesModelSettings link it'll take them to the docs that show the 3 different properties.